internal/notifier/alertmanager.go

```diff
@@ -28,12 +28,14 @@ import (
 	"golang.org/x/text/language"
 
 	eventv1 "github.com/fluxcd/pkg/apis/event/v1beta1"
+	"github.com/hashicorp/go-retryablehttp"
```

Comment:
Please put this import on the block right above. This is the order of our import blocks:

1. Built-in Go packages
2. Third-party packages
3. Flux packages

`"github.com/hashicorp/go-retryablehttp"` is a third-party.

Also, running `go fmt` would put it right on top of the other two third-party packages (`"golang.org/x/text/cases"` and `"golang.org/x/text/language"`).